frontend/packages/network-attachment-definition-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash';
+import { NetworkAttachmentDefinitionConfig, NetworkAttachmentDefinitionKind } from '../types';
+
+export const getConfig = (
+  obj: NetworkAttachmentDefinitionKind,
+): NetworkAttachmentDefinitionConfig => JSON.parse(_.get(obj, 'spec.config'));
```

Comment:
Please add handling parser error.

Out of scope for this PR: The json string in `NetworkAttachmentDefinition` structure is anti-pattern.

Reply:
Error handling added!

I agree with the sentiment regarding the JSON string. It definitely wouldn't have been my first choice.